### ISSUE TITLE
Application UI Follow Up and Changes

### ISSUE THREAD

**Issue by cschieder** [kubermatic/dashboard#4914]
### General
- Allow users to edit application names 
- ~Fix table (it doesn't span full width of the card)~ (Fixed in https://github.com/kubermatic/dashboard/pull/5936)
![table wrong](https://user-images.githubusercontent.com/49145827/188568455-7f7e3068-563e-4523-b6c7-d6a3b1e7077c.PNG)

### Logos
Allow User to add and display logos

#### Design, in practise if a logo is provided it would replace the application
https://www.figma.com/file/dJQKgUPzIDND36o2Nq8dct/Loodse-UI?node-id=7662%3A84466
![image](https://user-images.githubusercontent.com/49145827/188565324-68236a7b-f2b2-4c38-b9c8-5fe285df35c3.png)
![image](https://user-images.githubusercontent.com/49145827/188565357-92e71a5f-5f2b-44c2-80f0-bb10bbb797e1.png)
![image](https://user-images.githubusercontent.com/49145827/188565420-44d58d46-063a-4058-8ea2-cdbdf7c38307.png)
![image](https://user-images.githubusercontent.com/49145827/188565459-85bbf835-0e2a-4cb4-aeb4-47be275923f0.png)

#### Conditions
- if no logo is provided display application name
- provide documentation that includes what logos should be used (mark + text, horizontal format, should work in darkmode, maybe make it possible to provide 2 logos, one for light and one for dark mode) 
- allow custom sizing for each logo
- add max-height: 24px 

### Rough Documentation draft with placeholders
To add a logo [step by step instructions]

For best results, the logo should look like this
![grafanaaa](https://user-images.githubusercontent.com/49145827/188567422-8c7e60ab-8518-4c0b-b8d7-266d76030c5c.png)

In other words
- horizontal format
- shows both logo and name of the application
- application name is large and readable 
- is transparent (to work on darkmode) OR has an additional dark mode version
- Format: SVG or PNG

When you're done, check the logo in the UI and add custom sizing if needed
[Add screenshot of UI after the changes]

affects #4663
